src/styles/blocks/top-bar.scss

```diff
@@ -0,0 +1,43 @@
+.top-bar {
+  box-sizing: border-box;
+  padding-block: 24px;
+  justify-content: space-between;
+  align-items: center;
+
+  @include page-grid;
+
+  &__logo {
+    grid-column: 1;
+    display: flex;
+
+    @include hover(scale, 1.1);
+    
+    @include on-tablet {
+      padding-block: 11px;
+    }
+  }
+
+  &__button {
+    display: none;
+
+    @include on-tablet {
+      display: flex;
+      align-items: center;
+      gap: 30px;
+    }
+  }
+
+  &__buttons-holder {
+    grid-column-end: -1;
```

Comment:
Try to adjust this on tablet size to -2. The top bar is very close to right side of the page.
![image](https://github.com/mate-academy/Kickstarter/assets/37789845/a0d67369-6e0c-408d-9709-3ad1c2e5c68c)